package.json

```diff
@@ -42,6 +43,7 @@
     "last 3 edge versions"
   ],
   "devDependencies": {
+    "@solid/community-server": "^5.1.0",
```

Comment:
While I think we need to swap over from Node Solid Server to Community Solid Server, I'm not too sure about having it integrated with PASS directly.

I would need some opinion from @k-mack. Do you think this would be something that we want for PASS?

Reply:
I'm not sure I understand what you mean. Maybe I wasn't clear about what this does. This is just a dev tool to make development easier for engineers. Instead of having to test PASS on a remote server (something they don't have control over), this just makes it convenient for them to run their own local server, that they do have control over. It doesn't integrate with PASS directly.

I agree that PASS shouldn't have any specific integration with a particular type of server. It's a client application, and should be able to run against any valid SOLID server. It doesn't (and shouldn't) have knowledge of the specifics of the server implementation it's calling.

Reply:
Sorry @timbot1789, I think I might be a bit confused too on my end. Please correct me if I'm wrong on this.

You're trying to do now is to allow devs to bypass the existing Solid Pods on opencommons.net and solidcommunity.net by trying to create the Pod locally at localhost:3000, correct? So that way, we can have some mock server functionality to log in and work on the rest of the application or allows us to bypass log in completely and just move onto the other pages to work on them.

Reply:
Never mind, it was a misunderstanding on my part.